types/video.js/index.d.ts

```diff
@@ -2042,11 +2042,51 @@ declare namespace videojs {
 
     type Content = string | Element | Node | (() => string | Element | Node);
 
+    interface PlaybackRateMenuButton extends Component {
+        /**
+         * Returns the HTML Div Element
+         *
+         * @return the HTML Div Element.
+         */
+        el(): HTMLDivElement;
+    }
+
+    interface LiveDisplay extends Component {
+        /**
+         * Returns the HTML Div Element
+         *
+         * @return the HTML Div Element.
+         */
+        el(): HTMLDivElement;
+    }
+
+    interface ProgressControl extends Component {
+        /**
+         * Returns the HTML Div Element
+         *
+         * @return the HTML Div Element.
+         */
+        el(): HTMLDivElement;
+    }
+
+    interface RemainingTimeDisplay extends Component {
+        /**
+         * Returns the HTML Div Element
+         *
+         * @return the HTML Div Element.
+         */
+        el(): HTMLDivElement;
+    }
+
     /**
      * Container of main controls.
      */
     interface ControlBar extends Component {
+        liveDisplay: LiveDisplay;
```

Comment:
Accessing children as properties of ControlBar seems to be an undocumented behavior: https://docs.videojs.com/controlbar

Child components should instead be accessed by getChild(name). If you are looking for a typed way of retrieving child components consider adding overrides of getChild(name)?

Reply:
Alright, I created a new interface for ControlBar children. So I overload the method getChild with `HTMLDivElement`.